spras/evaluation.py

```diff
@@ -108,38 +112,117 @@ def precision_and_recall(file_paths: Iterable[Path], node_table: pd.DataFrame, a
             results.append({"Pathway": f, "Precision": precision, "Recall": recall})
 
         pr_df = pd.DataFrame(results)
+        return pr_df
+
+    @staticmethod
+    def visulize_precision_and_recall_per_pathway(pr_df: pd.DataFrame, output_file: str, output_png: str, aggregate_per_algorithm: bool = False):
```

Comment:
make a forth function to make the visualization and pass in the title. Keep the other two functions ot catch the edge cases.